modules/ROOT/pages/6.0-release-notes.adoc

```diff
@@ -0,0 +1,37 @@
+= TinyMCE 6.0 Release Notes
```

Comment:
I don't think this page makes sense? We don't want a monolith page so this should be removed as we already have the smaller pages.